src/styles/styles.js

```diff
@@ -2761,6 +2761,20 @@ const styles = {
         fontSize: variables.fontSizeSmall,
         ...spacing.ph2,
     },
+
+    workspaceSection: {
+        backgroundColor: colors.gray1,
+        borderRadius: variables.componentBorderRadiusCard,
+        marginVertical: 10,
```

Comment:
Maybe we can just do bottom margin for all of these cards? This way the first card in a view will sit more towards the top.

Reply:
oh that works